pkg/provider/doc_helpers.go

```diff
@@ -0,0 +1,24 @@
+package provider
```

Comment:
I think this should be in `internal/provider` package (as long as it will be still usable from the doc generator)

Reply:
I'll try and we'll see

Reply:
Done (moved to internal/provider/docs)